var/spack/repos/builtin/packages/verilator/package.py

```diff
@@ -75,18 +75,20 @@ class Verilator(AutotoolsPackage):
     depends_on("libtool", type="build")
     depends_on("help2man", type="build")
     depends_on("bison", type="build")
-    depends_on("flex")
-    depends_on("ccache", type=("build", "run"), when="@5.018:")
+    depends_on("flex", type=("build", "link"))
```

Comment:
Not opposed to having this explicitly stated, but `depends_on()` already defaults to `("build", "link")`.
https://spack.readthedocs.io/en/latest/packaging_guide.html#dependency-types

Reply:
ah yes, I think i went around that a while back and forgot.  I needed to see if it failed using just build and passed with build and link.  But I am not precious, I'll roll it back